lib/contacts.py

```diff
@@ -54,15 +55,23 @@ def import_file(self, path):
         try:
             with open(path, 'r') as f:
                 d = self._validate(json.loads(f.read()))
-        except json.decoder.JSONDecodeError:
+        except ValueError:
```

Comment:
What's the reason for this change?

Reply:
https://stackoverflow.com/a/44714607/672396

Reply:
Ok. Can you add a comment that reflects this? like `# backwards compatibility for JSONDecodeError ` or similar